examples/css/index.html

```diff
@@ -935,6 +935,103 @@ <h4 class="mistica-text-title1">3 + 9</h4>
                         <div class="placeholder" style="height: 120px; width: auto"></div>
                     </div>
                 </section>
+                <h3 class="mistica-text-title2">Padding</h3>
+                <div class="mistica-padding-8" data-demo-container>
```

Comment:
why not call it "box"? Suppose because padding is more easy to understand.. but just to know why this misalign with react lib

Reply:
because we need padding-x, padding-y padding-top, padding-bottom, etc. How do you do that with box? remember we don't have props in css like with react components